ts/components/registration/stages/CreateAccount.tsx

```diff
@@ -97,10 +93,6 @@ export const CreateAccount = () => {
     }
 
     try {
-      if (!privateKeyBytes) {
-        throw new Error('Private key not found');
-      }
```

Comment:
you've changed this so all good, but just so you know an empty array is truthy
![image](https://github.com/user-attachments/assets/3f972958-8d4c-4152-87f3-dfefcebcbc16)

Reply:
Yup I know I probably messed up here previously